### PR TITLE
Draft Legal Terms for new Kubernetes Clusters

### PR DESCRIPTION
**Pull request by othylmann**
there are still some comments left but I want to store this safely :)

### CODE REVIEW

File: limits.md [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
 
-* an unlimited number of services
-* an unlimited number of components per service
+However, we suggest you contact us before going over the following limits:
 
-However, we enforce these restrictions:
+* 5 nodes per cluster
+* 32GB RAM per node
+* 50 pods per node
+* persistent storage volume over 100GB per volume
```
Comment (marians) [inline limits.md]: "over 100GB"? Or "up to 100GB"?

Comment (othylmann) [inline limits.md]: Well, I would say "over 100GB" as only then they should tell us. But actually it can be "at"

Comment (marians) [inline limits.md]: Reading the contxt again, I suggest to change it to "persistent storage of 100GB per volume"

File: privacypolicy.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Giant Swarm collects personal data only when you provide it to us, through regis
...
+The personal data Giant Swarm collects will only be used for the purpose of supplying you with the requested services or for other purposes for which you have given your consent, unless Giant Swarm is legally required to do so. Giant Swarm uses credit card or bank details information solely to check the financial qualifications and to collect payment for the services of Giant Swarm. Credit Card information is never stored or even touched by Giant Swarm but handled directly by a qualified third party that handles the billing process. Giant Swarm also uses your e-mail address in order to offer you similar services of our own. You may object to this use of your e-mail address at any time without incurring any costs simply by clicking on the link at the end of our e-mails or send your objection to the following e-mail address [support@giantswarm.io](mailto:support@giantswarm.io).
```
Comment (marians) [inline privacypolicy.md]: I suggest to remove `or even touched`, since it's ambigious and not meaningful IMHO.

File: sla.md [hunk 3]
```diff
@@ -12,21 +12,34 @@ Generally the definitions as made and used in the TOS apply.
...
 - "Monthly Uptime Percentage" is calculated by subtracting from 100% the percentage of minutes during the month in which the Giant Swarm Service, as applicable, was in the state of "GS Unavailable." Monthly Uptime Percentage measurements exclude downtime resulting directly or indirectly from any GS SLA Exclusion (defined below).
 
-- "Shared Cluster" is a system where you are running containers without exclusive hardware and virtual servers to power them but on a shared hardware.
+- “Kubernetes Master” is the controlling unit(s) of your Kubernetes Cluster.
 
-- "Private Cluster" is a system of hardware or virtual servers available in your cluster.
+- “Nodes” are your compute instances of your Kubernetes Cluster.
 
-- "Cluster Storage" is our persistent facility.
+- "Cluster Storage" is our persistent storage facility.
```
Comment (marians) [inline sla.md]: We decided elsewhere to call this "persistent storage". Could be defined like this:

"Persistent Storage" is our facility to provide you with long-term data storage.

File: terms.md [hunk 4]
```diff
@@ -10,13 +10,13 @@
...
 
 ## 2. Giant Swarm Account
 
-2.1 In order to access some features or main elements of the Giant Swarm Service, you will have to create a Giant Swarm Account (hereinafter "Account").
+2.1 In order to access some features or main elements of the Giant Swarm Service, you will have to create a Giant Swarm Account compromised of your personal Login details and an Organization running one or several clusters. (hereinafter "Account").
```
Comment (marians) [inline terms.md]: compromised -> comprised

File: terms.md [hunk 5]
```diff
@@ -10,13 +10,13 @@
...
 
-2.2 When creating your Account, you must provide current, accurate and complete identification, contact and other information. You are responsible to maintain, keep current and update any registration data and other information you provide to Giant Swarm.
+2.2 When creating your Account , you must provide current, accurate and complete identification, contact and other information. You are responsible to maintain, keep current and update any registration data and other information you provide to Giant Swarm.
```
Comment (marians) [inline terms.md]: "Account ,": that blank character before the comma shouldn't be there

**Comment by marians** [comment]
@othylmann I'm done with my review.

File: privacypolicy.md (hunk 2 above)
Comment (puja108) [inline privacypolicy.md]: maybe `or even read`

File: sla.md [hunk 6]
```diff
@@ -12,21 +12,34 @@ Generally the definitions as made and used in the TOS apply.
 
 - "Monthly Uptime Percentage" is calculated by subtracting from 100% the percentage of minutes during the month in which the Giant Swarm Service, as applicable, was in the state of "GS Unavailable." Monthly Uptime Percentage measurements exclude downtime resulting directly or indirectly from any GS SLA Exclusion (defined below).
 
-- "Shared Cluster" is a system where you are running containers without exclusive hardware and virtual servers to power them but on a shared hardware.
+- “Kubernetes Master” is the controlling unit(s) of your Kubernetes Cluster.
 
-- "Private Cluster" is a system of hardware or virtual servers available in your cluster.
+- “Nodes” are your compute instances of your Kubernetes Cluster.
```
Comment (puja108) [inline sla.md]: change first `your` to `the` otherwise it's 2 `your`s

File: sla.md [hunk 7]
```diff
@@ -56,4 +70,4 @@ d) your request logs that document the errors and corroborate your claimed outag
...
...ster Storage, or any other Kubernetes Master or Kubelet or Cluster Storage performance issues: (i) that result from a suspension described in Section 4.1 of the TOS; (ii) caused by factors outside of our reasonable control, including any force majeure event or Internet access or related problems beyond the demarcation point of Shared Cluster or Private Cluster or Cluster Storage; (iii) that result from any actions or inactions of you or any third party; (iv) that result from your equipment, software or other technology and/or third party equipment, software or other technology (other than third party equipment within our direct control); (v) that result from failures of individual instances or volumes not attributable to GS Unavailability; (vi) that result from any maintenance as provided for pursuant to the Giant Swarm Terms; or (vii) arising from our suspension and termination of your right to use Giant Swarm Services in accordance with the Giant Swarm Terms (collectively, the "GS SLA Exclusion"). If availability is impacted by factors other than those used in our Monthly Uptime Percentage calculation, then we may issue a Service Credit considering such factors at our discretion.
```
Comment (puja108) [inline sla.md]: There's mentioning of `Shared Cluster or Private Cluster` in  (ii)

**Comment by puja108** [comment]
other than the few comments (also Marian's), LGTM

File: limits.md (hunk 1 above)
Comment (othylmann) [inline limits.md]: changed.

File: privacypolicy.md (hunk 2 above)
Comment (othylmann) [inline privacypolicy.md]: changed to even read

File: sla.md (hunk 3 above)
Comment (othylmann) [inline sla.md]: - "Cluster Storage" is our persistent storage facility, providing you with long-term data storage.

File: terms.md (hunk 4 above)
Comment (othylmann) [inline terms.md]: done

File: terms.md (hunk 5 above)
Comment (othylmann) [inline terms.md]: done

File: sla.md (hunk 7 above)
Comment (othylmann) [inline sla.md]: good catch 

point of the Kubernetes Master, Nodes or Cluster Storage

**Comment by marians** [comment]
What have I done? That wasn't intended. :( Clicked the wrong buttons in GitHub for Mac.

**Pull request merged**